docs/user/convert/parsing_and_validation.rst

```diff
@@ -191,17 +191,26 @@ through the converter evaluation context, or can be exposed through reporters co
       }
     }
 
-Reporters are available for `SLF4J <https://www.slf4j.org/>`__, `Graphite <https://graphiteapp.org/>`__ and
-`Ganglia <http://ganglia.sourceforge.net/>`__. To use Graphite or Ganglia, you must include a dependency on
-``geomesa-convert-metrics-graphite_2.11`` or ``geomesa-convert-metrics-ganglia_2.11``, respectively. Note that
-using Ganglia requires an additional GPL-licensed dependency ``info.ganglia.gmetric4j:gmetric4j``, which is excluded
-by default. The reporters can be configured as follows:
+Reporters are available for `SLF4J <https://www.slf4j.org/>`__, `CloudWatch <https://aws.amazon.com/cloudwatch/>`__,
+`Graphite <https://graphiteapp.org/>`__, and `Ganglia <http://ganglia.sourceforge.net/>`__. To use CloudWatch,
+Graphite, or Ganglia, you must include a dependency on ``geomesa-convert-metrics-cloudwatch_2.11``, 
+``geomesa-convert-metrics-graphite_2.11``, or ``geomesa-convert-metrics-ganglia_2.11``, respectively. 
+Note that using Ganglia requires an additional GPL-licensed dependency ``info.ganglia.gmetric4j:gmetric4j``, 
+which is excluded by default. The reporters can be configured as follows:
 
 ::
 
     geomesa.converters.myconverter {
       options {
         reporters = [
+          {
+            type        = "cloudwatch"
+            units       = "MILLISECONDS"
+            interval    = "60 seconds"
+            namespace   = "geomesa"
+            raw-counts  = false
+            zero-values = true
```

Comment:
How does the code authenticate with AWS CloudWatch to post metrics?  How is any of that configured?

Reply:
It uses your default credentials and region. Should I make that configurable?

Reply:
What do you mean by 'default credentials'?  Where do those defaults come from?

Reply:
i think .aws/credentials

Reply:
Hmm... there can be profiles in there...  How does those interact?  Also, if we are pulling from default locations, it is worth documenting that.

Reply:
One of the credentials in there should be listed as default, and I could make it so you have to configure your credentials in the converter if that is preferable.